src/generator/_generator.scss

```diff
@@ -2,6 +2,19 @@
 
 /// Only run if there should be an output
 @if ($outputTokenCSS == true) {
+  $design-tokens:   map-get($gorko-config, 'design-tokens');
```

Comment:
I think you have one of those auto-spacing things going one here, but let's keep it as a single space.